setup.py

```diff
@@ -43,8 +43,8 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.16",
-        "qujax ~= 0.3.4",
+        "pytket ~= 1.18",
+        "qujax ~= 1.0.0",
```

Comment:
Can we use `~= 1.0`? This ought to be enough to ensure compatibility according to semver.

Reply:
Good point, done in @cqc-melf
[change 1.0.0. to 1.0](https://github.com/CQCL/pytket-qujax/pull/94/commits/8d42a867af2269ba6d06ebc43ded6bd8151ef892)
[8d42a86](https://github.com/CQCL/pytket-qujax/pull/94/commits/8d42a867af2269ba6d06ebc43ded6bd8151ef892)